content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Groups-based authorization and authorization for list-typed claims
+description: Tutorial on how to configure the groups-base authorization and configure the authorization of list-typed claims in Istio.
+weight: 10
+keywords: [security,authorization]
+---
+
+This task go through examples of configuring the groups-base authorization and
```

Comment:
Grammar and word choice:
This tutorial walks you through examples to configure the groups-base authorization and the authorization of list-typed claims in Istio.

Reply:
Done.